frontend/language/src/nb.json

```diff
@@ -108,6 +111,9 @@
   "app_release.release_title": "Appen bygges basert på",
   "app_release.release_title_link": "siste endringstransaksjon til master",
   "app_release.release_version": "Versjon:",
+  "code_list_editor.duplicate_values_error": "Verdien er ikke unik",
+  "code_list_editor.empty": "Kodelisten er tom",
+  "code_list_editor.general_error": "Kodelisten har ugyldig format",
```

Comment:
I think Gørild and Dorte now have decided that validation/error messages should end with a period.

Reply:
Yup! [This PR](https://github.com/Altinn/altinn-studio/pull/14051/files#diff-3c337bd77f5ff70f1f68d29294960afca38ea082fe9b2716b109a46d89fdf8ddR119-R121) will fix all these texts 🫡

Reply:
Should of course operated with PR stacking here 😉 But will rebase this PR with main when the other PR is merged, before we merge this one!

Reply:
Nice 🙌